.all-contributorsrc

```diff
@@ -84,6 +84,15 @@
         "code"
       ]
     },
+    {
+      "login": "ashutosh1919",
+      "name": "Ashutosh Hathidara",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/20843596?s=460&u=f9348a38571fa0993d58db1c06d471b53210dc74&v=4",
+      "profile": "https://ashutosh1919.github.io",
+      "contributions": [
+        "code"
+      ]
+    },
```

Comment:
This is all-contributor bot's job so your PR shouldn't change these files as this isn't directly feature related.
Can you remove this?